viewer/test/viewer-test-pptr.js

```diff
@@ -251,6 +265,30 @@ describe('Lighthouse Viewer', () => {
     });
   });
 
+  describe.only('Renders old reports', () => {
+    [
+      'lhr-3.0.0.json',
+      'lhr-4.3.0.json',
+      'lhr-5.0.0.json',
+      'lhr-6.0.0.json',
+      'lhr-8.5.0.json',
+    ].forEach((testFilename) => {
+      it(`[${testFilename}] should load with no errors`, async () => {
+        await viewerPage.goto(viewerUrl, {waitUntil: 'networkidle2', timeout: 30000});
+        const fileInput = await viewerPage.$('#hidden-file-input');
+        const waitForAck = viewerPage.evaluate(() =>
+          new Promise(resolve =>
+            document.addEventListener('lh-file-upload-test-ack', resolve, {once: true})));
+        await fileInput.uploadFile(`${LH_ROOT}/report/test-assets/${testFilename}`);
```

Comment:
I don't think this promise structure guarantees that the event listener is installed by the time we call `fileInput.uploadFile`.

Reply:
Protocol commands are executed in sequence, so why wouldn't this be alright?

Reply:
> Protocol commands are executed in sequence

Interesting, did not know this. This is probably fine then.